classes/app/exhibition/ExhibitionBoundary.php

```diff
@@ -31,4 +31,22 @@ public function deleteExhibition(Delete\Request $request)
         $deleteExhibition = new Delete\DeleteExhibition($this->authenticator, $this->exhibitionRepository);
         $deleteExhibition->delete($request);
     }
+
+    public function listAllExhibitions(ListAll\Request $request)
+    {
+        $listAllExhibition = new ListAll\ListAllExhibitions($this->exhibitionRepository);
+        $response = new ListAll\Response();
```

Comment:
I think the use case itself should take care about creating a response object and set its values. The Flow of a use case is that it becomes a request as its input and returns an response as its output. Consequently, the boundary shouln't take care about that. In fact, the boundary is not more than a simple facade class which creates the corresponding use case class calls its public method with passing the incoming request and returning the response.